.github/workflows/ci.yml

```diff
@@ -50,6 +50,40 @@ jobs:
       ##  if: github.event_name == 'push' && github.ref == 'refs/heads/staging'
       #  run: echo "TEST_RUNS='[r1,r2,r3,r4,r5]'" | tee -a $GITHUB_ENV
 
+  generate_changelog:
```

Comment:
Looks like this was for testing.  Is this still needed?

Reply:
I suppose not, but this is an early check for the changelog rendering, even if it isn't final. I don't see the harm in having it run but we can disconnect it from any requirements to pass CI?

Reply:
Gotcha.  The only issue is that the workflow is duplicated.  Can you use a re-usable workflow?  https://docs.github.com/en/actions/using-workflows/reusing-workflows